Resources/Prototypes/_FarHorizons/Surgery/Robotics/robotics_steps.yml

```
@@ -0,0 +1,479 @@
+- type: entity
+  parent: SurgeryStepBase
+  id: SurgeryStepRemoveScrews
+  name: Remove Screws
+  components:
+  - type: SurgeryStep
+    tools:
+    - type: ScrewdriverSurgery
+  - type: Sprite
+    sprite: Objects/Tools/screwdriver.rsi
+    state: screwdriver-map
+  - type: OnFailDamage
+    damage: 
+      types:
+        Slash: 10
+
+- type: entity
+  parent: SurgeryStepBase
+  id: SurgeryStepRestoreScrews
+  name: Restore Screws
+  components:
+  - type: SurgeryStep
+    tools:
+    - type: ScrewdriverSurgery
+  - type: Sprite
+    sprite: Objects/Tools/screwdriver.rsi
+    state: screwdriver-map
+  - type: OnFailDamage
+    damage: 
+      types:
+        Slash: 10
+        
+
+- type: entity
+  parent: SurgeryStepBase
+  id: SurgeryStepRemoveInnerScrews
+  name: Remove Inner Screws
+  components:
+  - type: SurgeryStep
+    duration: 5
+    tools:
+    - type: ScrewdriverSurgery
+  - type: Sprite
+    sprite: Objects/Tools/screwdriver.rsi
+    state: screwdriver-map
+  - type: OnFailDamage
+    damage: 
+      types:
+        Slash: 10
+
+- type: entity
+  parent: SurgeryStepBase
+  id: SurgeryStepRestoreInnerScrews
+  name: Restore Inner Screws
+  components:
+  - type: SurgeryStep
+    duration: 4
+    tools:
+    - type: ScrewdriverSurgery
+  - type: Sprite
+    sprite: Objects/Tools/screwdriver.rsi
+    state: screwdriver-map
+  - type: OnFailDamage
+    damage: 
+      types:
+        Slash: 10
+
+- type: entity
+  parent: SurgeryStepBase
+  id: SurgeryStepExposeCoolantChannels
+  name: Expose Coolant Channels
+  components:
+  - type: SurgeryStep
+    duration: 4
+    tools:
+    - type: WirecutterSurgery
+  - type: Sprite
+    sprite: Objects\Tools\wirecutters.rsi
+    state: cutters-map
+  - type: OnFailDamage
+    damage: 
+      types:
+        Slash: 10
+
+- type: entity
+  parent: SurgeryStepBase
+  id: SurgeryStepConnectCoolantChannels
+  name: Connect Coolant Channels
+  components:
+  - type: SurgeryStep
+    duration: 5
+    tools:
+    - type: WrenchSurgery
+  - type: Sprite
+    sprite: Objects\Tools\wrench.rsi
+    state: icon
+  - type: OnFailDamage
+    damage: 
+      types:
+        Piercing: 10
+
+- type: entity
+  parent: SurgeryStepBase
+  id: SurgeryStepUnweldJoints
+  name: Unweld Joints
+  components:
+  - type: SurgeryStep
+    tools:
+    - type: WelderSurgery
+  - type: Sprite
+    sprite: Objects\Tools\welder.rsi
```

```suggestion
    sprite: Objects/Tools/welder.rsi
```
every instance of `\` needs to be changed into `/` in this file. It breaks on linux

---

please don't do it manually though, find and replace